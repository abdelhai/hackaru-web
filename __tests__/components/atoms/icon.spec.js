import Factory from '@/__tests__/__setups__/factory';
import Icon from '@/components/atoms/icon';

describe('Icon', () => {
  it('render correctly', () => {
    const wrapper = new Factory(Icon, {
      stubs: ['activity-icon'],
      propsData: {
        name: 'activity-icon'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
