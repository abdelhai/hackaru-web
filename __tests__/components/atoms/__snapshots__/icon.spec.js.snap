// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon render correctly 1`] = `
<activity-icon-stub
  class="icon"
/>
`;
